README.md

```diff
@@ -30,6 +30,33 @@ Additionally, the example code has been released to the public domain (see the
 Quick Start
 ===========
 
+### Windows
+
+You can use vcpkg install libgit2:
+
+```
+vcpkg install libgit2:x64-windows libgit2:x86-windows
+```
+
+### Ubuntu/Debian
+
+You can use apt/apt-get install libgit2
+
+```
+sudo apt install libgit2-dev
+```
+Unfortunately, the libgit2 installed in this way is not necessarily the latest version
```

Comment:
Proposal:

"""
The version of libgit2 installed depends on the distribution. As such, it is not guaranteed that you will get the latest version.
"""